WordPress/Classes/Stores/StatsInsightsStore.swift

```diff
@@ -0,0 +1,130 @@
+import Foundation
+import WordPressFlux
+import WordPressComStatsiOS
+
+enum InsightAction: Action {
+    case receivedLatestPostSummary(_ latestPostSummary: StatsLatestPostSummary?)
+    case refreshInsights()
+}
+
+enum InsightQuery {
+    case insights()
```

Comment:
This could be just `insights`, without the `()`, right?

Reply:
Thank you! Fixed.